contracts/contracts/Franklin.sol

```diff
@@ -730,10 +729,14 @@ contract Franklin {
     function payoutWithdrawNow(address _to, uint16 _tokenId, uint128 _amount) internal {
         if (_tokenId == 0) {
             address payable to = address(uint160(_to));
-            to.transfer(_amount);
-        } else if (_tokenId < governance.totalTokens() + 1) {
+            if (!to.send(_amount)) {
```

Comment:
where are the gas limits?

Reply:
where do we make sure the verification will always fit into block gas limit?

Reply:
AFAIK send gas stipend is always 2300
https://ethereum.stackexchange.com/questions/19341/address-send-vs-address-transfer-best-practice-usage

Reply:
> where do we make sure the verification will always fit into block gas limit?

rn we have constraint in the node